package/kernel/linux/modules/netdevices.mk

```diff
@@ -1141,3 +1141,22 @@ define KernelPackage/mlx5-core/description
 endef
 
 $(eval $(call KernelPackage,mlx5-core))
+
+
+define KernelPackage/sfp
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=SFP cage support
+  DEPENDS:=+kmod-i2c-core +kmod-hwmon-core
+  KCONFIG:= CONFIG_SFP \
+	CONFIG_MDIO_I2C=y
```

Comment:
Why is this `=y`?

Reply:
I am not sure, when do I need to add `=m`, `=y`, `=n` or just the name.
If I look in the file `netsupport.mk` I see many combination.

Making it a module, this feels like the right line. But maybe I am wrong.
```
  KCONFIG:= \
        CONFIG_SFP=m \
        CONFIG_MDIO_I2C=m
```

Reply:
`=y` means built-in
You can either use `=m` or just the name

Reply:
Fixed both with `=m`.